test/1.7_wrap_unwrap_uniV3.js

```diff
@@ -0,0 +1,213 @@
+const { Token } = require('@uniswap/sdk-core');
+const abi_uniswap = require('./abis/uniswapV3Pool.json');
+const abi_position_manager = require('./abis/positionManager.json');
+const poolAddress = "0x4e68Ccd3E89f51C3074ca5072bbAC773960dFa36";
+const poolImmutablesAbi = [
+  "function factory() external view returns (address)",
+  "function token0() external view returns (address)",
+  "function token1() external view returns (address)",
+  "function fee() external view returns (uint24)",
+  "function tickSpacing() external view returns (int24)",
+  "function maxLiquidityPerTick() external view returns (uint128)",
+];
+
+describe('Deploying the plexus contracts for WrapperUni adding liquidity test', () => {
+  let wrapper, owner;
+  let netinfo;
+  let network = 'unknown';
+  let tokenPairAddress = '';
+  let daiTokenAddress;
+  let sushiTokenAddress;
+  let compoundTokenAddress;
+  let wethAddress;
+  const unitAmount = "2";
+
+  // Deploy and setup the contracts
+  before(async () => {
+    const { deployedContracts } = await deployWrappersOnly();
+    wrapper = deployedContracts.wrapperUniV3;
+    owner = deployedContracts.owner;
+    addr1 = deployedContracts.addr1;
+    netinfo = await ethers.provider.getNetwork();
+    network = netinfo.chainId === 1 ? "mainnet" :
+      netinfo.chainId === 42 ? "kovan" :
+        netinfo.chainId === 56 ? "binance" :
+          netinfo.chainId === 137 ? "matic" : 'mainnet';
+    daiTokenAddress = addr.tokens.DAI[network];
+    usdtTokenAddress = addr.tokens.USDT[network];
+    usdcTokenAddress = addr.tokens.USDC[network];
+    farmTokenAddress = addr.tokens.FARM[network];
+    pickleTokenAddress = addr.tokens.PICKLE[network];
+    sushiTokenAddress = addr.tokens.SUSHI[network];
+    compoundTokenAddress = addr.tokens.COMP[network];
+    wethAddress = addr.tokens.WETH[network];
+  });
+
+  async function getPoolImmutables(poolContract) {
+    const PoolImmutables = {
+      factory: await poolContract.factory(),
+      token0: await poolContract.token0(),
+      token1: await poolContract.token1(),
+      fee: await poolContract.fee(),
+      tickSpacing: await poolContract.tickSpacing(),
+      maxLiquidityPerTick: await poolContract.maxLiquidityPerTick(),
+    };
+    console.log(PoolImmutables);
+    return PoolImmutables;
+  }
+
+  async function getPoolState(poolContract) {
+    const [liquidity, slot] = await Promise.all([
+      poolContract.liquidity(),
+      poolContract.slot0(),
+    ]);
+
+    const PoolState = {
+      liquidity,
+      sqrtPriceX96: slot[0],
+      tick: slot[1],
+      observationIndex: slot[2],
+      observationCardinality: slot[3],
+      observationCardinalityNext: slot[4],
+      feeProtocol: slot[5],
+      unlocked: slot[6],
+    };
+    console.log(PoolState);
+    return PoolState;
+  }
+
+
+  describe('Test Uni V2 liquidity pool', () => {
+
+    it('Should convert 2 ETH to USDT Token(s) from MakerDao via Uniswap', async () => {
```

Comment:
Please split this test to follow the same format of the other tests. So have 1 test section i.e. the `it` for wrapping and another one for `unwrapping`